modules/security-registries-quay.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-registries.adoc
+
+[id="security-registries-quay_{context}"]
+= Storing containers using Red Hat Quay
+link:https://access.redhat.com/products/red-hat-quay[Red Hat Quay] is an
+enterprise-quality container registry product from Red Hat.
+Development for Red Hat Quay is done through the upstream
+link:https://docs.projectquay.io/welcome.html[Project Quay].
+Red Hat Quay is available to deploy on-premise or through the hosted
+version of Red Hat Quay at link:https://quay.io[Quay.io].
+
+Security-related features of Red Hat Quay include:
+
+* *Time Machine*: Allows images with older tags to expire after a set
+period of time or based on a user-selected expiration time.
+
+* * link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html-single/manage_red_hat_quay/index#repo-mirroring-in-red-hat-quay[Repository mirroring]*: Lets you mirror
```

Comment:
The asterisks are rendering here. You just need to remove the space between the * and "link".

Reply:
Fixed.